libs/sdk-ui-filters/src/RankingFilter/types.ts

```diff
@@ -0,0 +1,23 @@
+// (C) 2020 GoodData Corporation
+import { Identifier, RankingFilterOperator } from "@gooddata/sdk-model";
+
+export interface IOperatorDropdownItem {
+    value: RankingFilterOperator;
+    translationId: string;
+}
+
+/**
+ * @beta
+ */
+export interface IAttributeDropdownItem {
+    title: string;
+    localIdentifier: Identifier;
```

Comment:
I guess something like this would work, with separate component for each type.

`enum AttributeType {
    DEFAULT,
    COMBINATION_OF_ALL,
    CUSTOM,
}

interface IAttributeDropdownItem {
    type: AttributeType;
    title?: string;
    localIdentifier?: Identifier;
}`